src/group/mls_group/mod.rs

```diff
@@ -288,11 +288,11 @@ impl MlsGroup {
     }
 
     /// Get the ciphersuite implementation used in this group.
-    pub(crate) fn ciphersuite(&self) -> &Ciphersuite {
```

Comment:
I think it's not a bad idea to expose the ciphersuite of a given group to the outside. For example, an application might want to figure out if any of its groups are using deprecated ciphersuites.